internal/controller/postgrescluster/pgbackrest.go

```diff
@@ -1634,6 +1649,9 @@ func (r *Reconciler) reconcilePostgresClusterDataSource(ctx context.Context,
 		return errors.WithStack(err)
 	}
 
+	// TODO(dsessler): If pgdata is being sourced by a VolumeSnapshot then don't perform a typical restore job;
```

Comment:
Can this TODO be deleted?

Reply:
It hasn't been implemented, so I figured it would stay...

Reply:
🌱 Rather than attach my name, I've started using topics in the parentheses. I'm not sure what word makes sense.

```suggestion
	// TODO(snapshots): If pgdata is being sourced by a VolumeSnapshot then don't perform a typical restore job;
```